spec/builtin-history.test.sh

```diff
@@ -195,4 +233,131 @@ status=1
 ## END
 
 
+#### HISTSIZE shrinks the history when changed
```

Comment:
Oh interesting, I didn't know this

This is similar to `LANG=C` - it actually makes a setlocale() syscall when the var is changed !!

I don't like that, because the syntax doesn't tell you that!  It's just a magic function call that looks like a variable!

Reply:
LANG is a rabbit hole I don't want to go down. All I know is it royally messed with my attempts to run the Sonarr/Radarr projects.